design/ybcluster_crd.yaml

```diff
@@ -0,0 +1,186 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: ybclusters.yugabytedb.yugabyte.com
+spec:
+  group: yugabytedb.yugabyte.com
+  names:
+    kind: YBCluster
+    listKind: YBClusterList
+    singular: ybcluster
+    plural: ybclusters
+  scope: Namespaced
+  version: v1alpha1
+  validation:
+    openAPIV3Schema:
```

Comment:
in case you're interested I think you can put [`description` blocks in these definitions](https://github.com/coreos/prometheus-operator/blob/master/example/prometheus-operator-crd/alertmanager.crd.yaml#L17-L19) so that anything you'd otherwise leave as a `# comment` could be defined and described in the schema itself?

Reply:
Sure. I might add some later.

Reply:
I added some descriptions in the CRD file.